README.md

```diff
@@ -29,9 +29,9 @@ irc.mozilla.org.
 
 ### Browser Extensions
 
-#### Firefox
+#### Firefox and Seamonkey
 
-PDF.js is built into version 19+ of Firefox, however one extension is still available:
+PDF.js is built into version 19+ of Firefox, however one extension is still available, in Seamonkey it needs to be manually installed:
```

Comment:
It's not clear what "it needs to be manually installed" mean. I don't think we need to add Seamonkey in Readme.md. However we can add description at https://github.com/mozilla/pdf.js/wiki/Frequently-Asked-Questions#faq-extensions if you explain installation process.